### DISCUSSION TITLE
What values ​​is this triangle really representing?

### DISCUSSION THREAD

**Discussion by DARIORAMON** [NightscoutFoundation/xDrip#1967] [Q&A]
Hello everyone, I'm new to xdrip, I wanted to ask about the green dotted line triangle.
What values ​​is this triangle really representing?
If someone could tell me I would be grateful. Greetings

**Comment by Navid200** [comment]
Can you post a screenshot?

**Reply by DARIORAMON** [reply]
Hi, yes of course, it's him; Green isosceles triangle with green points below. thank you
![Triangule ](https://user-images.githubusercontent.com/93885597/151678138-22715194-2f8f-48dc-adb8-74b95d7823bc.jpeg)

**Reply by Navid200** [reply]
I see a green arrow, green text, green curves, and green squares.
But, I see no green triangle.

Would you please edit the image and mark it?

**Comment by DARIORAMON** [comment]
Oh sorry, I must have a great imagination. It is the figure that I am indicating with the celestial date. Can you identify it now? Thanks
![Triangule2 ](https://user-images.githubusercontent.com/93885597/151678445-48a6af6b-b002-4cab-b25d-847132ac980f.jpg)

**Reply by Navid200** [reply]
You mean the curve.
You can see the legend here:
https://navid200.github.io/xDrip/docs/Legend.html

It tells you how to get to it.  You can modify those to find out which one it is.
I personally don't use the simulator.  So, am not sure exactly which item that curve represents.

**Reply by DARIORAMON** [reply]
Yes, I had already seen those references, and unfortunately they do not give much information about what it represents, they only give a name. Thank you very much for replying as well.

**Reply by Navid200** [reply]
You misunderstand.

Change the color of the items on that page one at a time until you find out which item corresponds to that curve.  You can reset all colors back to defaults after every try.
When you find out which item in the list corresponds to that curve, the next thing to find out is what that item represents.

Right now, you have two unknowns, which item on the list is it and what is it?
If you get an answer for the first question, we can look for an answer for the second question.
